components/edit-collective/sections/Tiers.js

```diff
@@ -232,6 +240,7 @@ class Tiers extends React.Component {
         options: collective =>
           getOptions(props.types || (collective.type === PROJECT ? SIMPLIFIED_TIER_TYPES : DEFAULT_TIER_TYPES)),
         label: intl.formatMessage(this.messages['type.label']),
+        disabled: props.types?.length === 1,
```

Comment:
@Betree : There's no point of making the select active when there's only one option (this is the case with tickets). Therefore I disabled this for one option and also renamed the `ticket (allow multiple tickets per order)` to just `Ticket` for clarity, now that we have the `singleTicket` switch.

Reply:
I'd go a bit further: why show type altogether in this case?

```suggestion
        when: props.types?.length !== 1,
```

Reply:
This one already have a `when` filed within the object and I think we need to check for whether the select options contains more than 1 option rather than just `props.types` here. So I've made the adjustments. Let me know how it looks now.